crates/agglayer-config/Cargo.toml

```diff
@@ -21,6 +21,8 @@ rstest.workspace = true
 serde_json = { workspace = true }
 toml.workspace = true
 
+agglayer-config = { path = ".", features = ["testutils"] }
```

Comment:
why do we need that?

Reply:
> why do we need that?

It failed to build if only building `agglayer-config` crate but it looks like it's no longer needed, so I removed it.